src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -443,5 +485,62 @@ private static byte GetSeverityNumber(LogLevel logLevel)
                 return 1;
         }
     }
+
+    private static int SanitizeCategoryName(byte[] buffer, int cursor, ref int validNameLength, string categoryName)
```

Comment:
Rename this method to indicate that it not only sanitizes the category name but also serializes it. For example, `SerializeSanitizedCategoryName`